etcdctlv3/command/range_command.go

```diff
@@ -48,13 +58,40 @@ func rangeCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+
+	sortByOrder := pb.RangeRequest_NONE
+	switch {
+	case sortOrder == "ASCEND":
+		sortByOrder = pb.RangeRequest_ASCEND
+	case sortOrder == "DESCEND":
+		sortByOrder = pb.RangeRequest_DESCEND
+	}
+
+	sortByTarget := pb.RangeRequest_KEY
+	switch {
+	case sortTarget == "CREATE":
```

Comment:
do we really want to differentiate upper and lower letters? Probably we can just do ToUpperLetters for the input flag?